lib/recurly/errors.php

```diff
@@ -7,7 +7,22 @@
  * @package    Recurly_Client_PHP
  * @copyright  Copyright (c) 2011 {@link http://recurly.com Recurly, Inc.}
  */
-class Recurly_Error extends Exception {}
+class Recurly_Error extends Exception {
+
+	private $recurlyCode;
+
+	public function __construct($message, $code = 0, Exception $previous = null, $recurlyCode = null)
```

Comment:
Why not just use the $code parameter? The class is `Recurly_Error` so it'd seem logical that the code would be a Recurly error code.